phoenix-core/src/main/java/org/apache/phoenix/schema/stats/StatisticsCollectorFactory.java

```diff
@@ -63,6 +64,10 @@ public static StatisticsCollector createStatisticsCollector(
         DISABLE_STATS.add(TableName.valueOf(PhoenixDatabaseMetaData.SYSTEM_FUNCTION_NAME));
         DISABLE_STATS.add(TableName.valueOf(PhoenixDatabaseMetaData.SYSTEM_SEQUENCE_NAME));
         DISABLE_STATS.add(TableName.valueOf(PhoenixDatabaseMetaData.SYSTEM_STATS_NAME));
+        DISABLE_STATS.add(SchemaUtil.getPhysicalTableName(PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME_BYTES,true));
```

Comment:
same as above, it just to avoid collecting stats for system table, so list contains both.